src/Symfony/Component/Cache/Traits/Relay/Relay12Trait.php

```diff
@@ -0,0 +1,107 @@
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits\Relay;
+
+if (version_compare(phpversion('relay'), '0.12.0', '>=')) {
+    /**
+     * @internal
+     */
+    trait Relay12Trait
+    {
+        public function delifeq($key, $value): \Relay\Relay|false|int
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->delifeq(...\func_get_args());
+        }
+
+        public function vadd($key, $values, $element, $options = null): \Relay\Relay|false|int
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vadd(...\func_get_args());
+        }
+
+        public function vcard($key): \Relay\Relay|false|int
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vcard(...\func_get_args());
+        }
+
+        public function vdim($key): \Relay\Relay|false|int
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vdim(...\func_get_args());
+        }
+
+        public function vemb($key, $element, $raw = false): \Relay\Relay|array|false
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vemb(...\func_get_args());
+        }
+
+        public function vgetattr($key, $element, $raw = false): \Relay\Relay|array|false|string
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vgetattr(...\func_get_args());
+        }
+
+        public function vinfo($key): \Relay\Relay|array|false
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vinfo(...\func_get_args());
+        }
+
+        public function vismember($key, $element): \Relay\Relay|bool
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vismember(...\func_get_args());
+        }
+
+        public function vlinks($key, $element, $withscores): \Relay\Relay|array|false
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vlinks(...\func_get_args());
+        }
+
+        public function vrandmember($key, $count = 0): \Relay\Relay|array|false|string
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vrandmember(...\func_get_args());
+        }
+
+        public function vrange($key, $min, $max, $count = -1): \Relay\Relay|array|false
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->vrange(...\func_get_args());
+        }
+
+        public function vrem($key, $element): \Relay\Relay|false|int
```

Comment:
@tillkruss Could it be that this method was forgotten to be mentioned in the changelog?